c/idcams.c

```diff
@@ -452,7 +452,6 @@ void idcamsPrintCommandOutput(const IDCAMSCommandOutput *idcamsOutput) {
 
   IDCAMSOutputRecord *currentNode = idcamsOutput->firstRecord;
   while (currentNode != NULL) {
-    printf("%.*s\n", currentNode->recordLength, currentNode->record);
```

Comment:
The only purpose of this function is to print debug information. Why did you remove this printf?